server/src/business-layer/services/StripeService.ts

```diff
@@ -259,4 +261,27 @@ export default class StripeService {
     /** Return the updated product */
     return updatedProduct
   }
+
+  public async getActiveMembershipProducts() {
```

Comment:
nit: should add jsdoc (see the other methods) so when someone hovers over the definition they don't have to guess what this does - in VS Code you can start this by typing `/*`